romm.wiki/Tinfoil-Integration.md

```diff
@@ -0,0 +1,62 @@
+This will help you configure Tinfoil integration for your switch to work with your RomM Library.
+
+- What in Tinfoil?
+- Prepare
+- Configure 
+- Additional
+- Enjoy!
+
+## What is [Tinfoil](https://tinfoil.io/)? 
+<img src="https://cdn2.steamgriddb.com/icon_thumb/1178cf1b6a47d41fc664b7d97e305840.png" alt="drawing" width="200"/>
+
+
+Glad you asked!
+
+Tinfoil is an applet that is installed directly to a "modded" Nintendo Switch. The application will let you connect to something called "shops". Shops are basically a hosted resource that will you stream and install certain formatted files directly to your Nintendo Switch!
```

Comment:
remove this entire section, i don't want nintendo to sue us lol
instead just link to it in the first sentence like

```
This will help you configure the [Tinfoil](https://tinfoil.io/) integration for your Switch to work with your RomM library.
```